index.js

```diff
@@ -1,4 +1,8 @@
 
+const EPSILON = Math.pow(2, -52);
+const EPSILON_IN_CIRCLE = 12 * EPSILON;
```

Comment:
in Shewchuk's code robustness limit for incircle is calculated like this:
```
epsilon = DBL_EPSILON;
iccerrboundA = (10.0 + 96.0 * epsilon) * epsilon;
permanent = (Absolute(bdxcdy) + Absolute(cdxbdy)) * alift
            + (Absolute(cdxady) + Absolute(adxcdy)) * blift
            + (Absolute(adxbdy) + Absolute(bdxady)) * clift;
  errbound = iccerrboundA * permanent;
  if ((det > errbound) || (-det > errbound)) {
    return det;
  }
```

Reply:
I don't think we have to go that far. I took mine here https://github.com/JuliaGeometry/GeometricalPredicates.jl/blob/master/src/GeometricalPredicates.jl#L48